src/OpenDict.cc

```diff
@@ -0,0 +1,950 @@
+// See the file "COPYING" in the main distribution directory for copyright.
+
+#include "zeek-config.h"
+
+#ifdef HAVE_MEMORY_H
+#include <memory.h>
+#endif
+#include <algorithm>
+#include <signal.h>
+#include <climits>
+#include <fstream>
+#include "OpenDict.h"
+#include "Reporter.h"
+#include "util.h"
+
+#ifdef DEBUG
+#define ASSERT_VALID(o)	o->AssertValid()
+#else
+#define ASSERT_VALID(o)
+#endif//DEBUG
+
+struct IterCookie {
```

Comment:
This was forward-declared as a `class` in the header. This should change to match. Make sure to add a `public:` access specifier when you change it.